### PR TITLE
Fix URLs for PageWaitForSelectorOptions 

### PR DESCRIPTION
**Pull request by dgabriel123**
#42 Fixed 7 instances of the PageWaitForSelectorOptions URL to https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ await expect(page).toHaveText("#foo", "my text")
 
 ### toHaveSelector
 
-**expect(page: [Page]).toHaveSelector(selector: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [Page]).toHaveSelector(selector: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:60]:
```suggestion
**expect(page: [Page]).toHaveSelector(selector: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

File: README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ This function checks if the the `textContent` of a given element contains the pr
 
 You can do this via a selector on the whole page:
 
-**expect(page: [Page]).toHaveText(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [Page]).toHaveText(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:78]:
```suggestion
**expect(page: [Page]).toHaveText(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

File: README.md [hunk 3]
```diff
@@ -106,23 +106,23 @@ This function checks if the `textContent` of a given element is the same as the
 
 You can do this via a selector on the whole page:
 
-**expect(page: [Page]).toEqualText(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [Page]).toEqualText(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:109]:
```suggestion
**expect(page: [Page]).toEqualText(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

File: README.md [hunk 4]
```diff
@@ -106,23 +106,23 @@ This function checks if the `textContent` of a given element is the same as the
...
 
 ```javascript
 await expect(page).toEqualText("#my-element", "Playwright")
 ```
 
 Or without a selector which will use the `body` element:
 
-**expect(page: [Page]).toEqualText(value: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [Page]).toEqualText(value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:117]:
```suggestion
**expect(page: [Page]).toEqualText(value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

File: README.md [hunk 5]
```diff
@@ -106,23 +106,23 @@ This function checks if the `textContent` of a given element is the same as the
...
 
 ```javascript
 await expect(page).toEqualText("Playwright")
 ```
 
 Or by passing a Playwright [ElementHandle]:
 
-**expect(page: [ElementHandle]).toEqualText(value: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [ElementHandle]).toEqualText(value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:125]:
```suggestion
**expect(page: [ElementHandle]).toEqualText(value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

File: README.md [hunk 6]
```diff
@@ -137,15 +137,15 @@ This function checks if the `value` of a given element is the same as the provid
 
 You can do this via a selector or the element directly:
 
-**expect(page: [Page]).toEqualValue(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [Page]).toEqualValue(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:140]:
```suggestion
**expect(page: [Page]).toEqualValue(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

**Review by mxschmitt — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -137,15 +137,15 @@ This function checks if the `value` of a given element is the same as the provid
...
+**expect(page: [Page]).toEqualValue(selector: string, value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
 
 ```javascript
 await expect(page).toEqualValue("#my-element", "Playwright")
 ```
 
 Or by passing a Playwright [ElementHandle]:
 
-**expect(page: [ElementHandle]).toEqualValue(value: string, options?: [PageWaitForSelectorOptions](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options))**
+**expect(page: [ElementHandle]).toEqualValue(value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/?_highlight=aitforselector#pagewaitforselectorselector-options))**
```
Comment (mxschmitt) [inline README.md:148]:
```suggestion
**expect(page: [ElementHandle]).toEqualValue(value: string, options?: [PageWaitForSelectorOptions](https://playwright.dev/docs/api/class-page/#pagewaitforselectorselector-options))**
```

**Comment by mxschmitt** [comment]
Thanks!

**Comment by dgabriel123** [comment]
> Thanks!

Спасибо!

**Pull request merged**